posts/a-short-history-of-code-search.md

```diff
@@ -43,17 +43,17 @@ Linux, too, would benefit from search innovation as early as 1994. Two Norwegian
 
-Also built on Apache Lucene is [OpenGrok](https://oracle.github.io/opengrok/), which is an open-source project built in Java and maintained by Oracle. Originally written by Chandan B.N., OpenGrok offers fast code search as well as a cross reference engine allowing developers to navigate their source tree. OpenGrok is very popular and actively maintained. 
+Also built on Apache Lucene is [OpenGrok](https://oracle.github.io/opengrok/), which is an open-source project written in Java and maintained by Oracle. Originally developed by [Chandan B.N.](https://github.com/oracle/opengrok/wiki/Story-of-OpenGrok), OpenGrok offers fast code search as well as a cross reference engine allowing developers to navigate their source tree. OpenGrok is very popular and actively maintained today. 
 
 ## The future of code search 
 
-Google has had an internal code search tool that has long been considered a gold standard of code search. The work on the internal tool began with gsearch by Jeff Dean in the early 2000s and is widely considered to be a productivity boost that also makes large-scale refactoring easier. In 2010, Steve Yegge led the development of Grok at Google for code comprehension, and a version of Grok was eventually open-sourced as Kythe in 2015 and is actively under development. Google Code Search, led by Russ Cox, was a free beta public product that debuted in Google Labs in 2006 but was shut down in 2011. There currently is a [publicly re-released](https://cs.opensource.google/) version that debuted in 2020 for use exclusively with Google open source software and Google Cloud Source Repositories.
+There are many code search efforts today in active development that gesture toward what developers may expect from code search in the future. Some of these include closed source tools like Google’s internal search tool, making its large code base more accessible to those working on it. The work on Google’s code search began with gsearch by Jeff Dean in the early 2000s, which offered a productivity boost for developers and made large-scale refactoring easier. In 2010, Steve Yegge led the development of Grok at Google for code comprehension; a version of Grok was eventually open-sourced as Kythe in 2015 and is actively under development. Google Code Search, led by Russ Cox, was a free beta public product that debuted in Google Labs in 2006 but was shut down in 2011. There currently is a [publicly re-released](https://cs.opensource.google/) version that debuted in 2020 for use exclusively with Google open source software and Google Cloud Source Repositories.
```

Comment:
Added an introductory sentence to this section to make it flow better, and some clarification around Google's tool (also made it a little less subjective towards Google)